### ISSUE TITLE
"Collapse all" in System Configuration menu

### ISSUE THREAD

**Issue by krisgry** [LSTS/neptus#85]
Hi,
A really nice feature for the System Configuration menu would be to add an option to "collapse all" the subcategories (i.e. task) of parameters. This list of parameters gets really long when "Access developer parameters" is activated, which makes for a lot of scrolling when changing parameters in different tasks. A solution to this, of course, is to manually collapse the parameters from tasks that you are not currently doing changes in, but this is time consuming and has to be done each time you restart DUNE (because of changes in the parameters)(especially frustrating when trying different parameters in SITL).

TL;DR: I would like **one** button that made the System Configuration menu collapse to something like this:
![image](https://user-images.githubusercontent.com/1384876/35221065-156ccda8-ff79-11e7-88b1-1c457adc2e86.png)

I know that Joao Fortuna has asked about this before, so this is just posted to have a public place to track the progress.

**Comment by zepinto** [comment]
Fixed in lsts/neptus@42eb7d96b606dbd2dc4d93af0ea0b82942de14d0.
